presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -504,7 +504,7 @@ private static OrcPredicate getPredicate(TupleDomain<RaptorColumnHandle> effecti
                 columns.add(new ColumnReference<>(column, index, column.getColumnType()));
             }
         }
-        return new TupleDomainOrcPredicate<>(effectivePredicate, columns.build());
+        return new TupleDomainOrcPredicate<>(effectivePredicate, columns.build(), true);
```

Comment:
Let's set this to false for now.  The ORC writer does not create files with bloom filters right now anyway.